federationapi/storage/shared/storage_edus.go

```diff
@@ -33,13 +41,20 @@ func (d *Database) AssociateEDUWithDestination(
 	receipt *Receipt,
 	eduType string,
 ) error {
+	var expiresAt *gomatrixserverlib.Timestamp
+	if duration, ok := expireEDUTypes[eduType]; ok {
```

Comment:
I think all EDUs should probably expire given they are ephemeral. Maybe default to 24h or something?